perf/security/file-mount-tests/non-sds-2/setup_test.sh

```diff
@@ -12,18 +12,32 @@ if [[ ! -d "${WD}" ]]; then
 fi
 
 URL=""
-if [[ "$RELEASETYPE" == "daily" ]]; then
-  URL="https://gcsweb.istio.io/gcs/istio-prerelease/daily-build/${RELEASE}/istio-${RELEASE}-linux.tar.gz"
-elif [[ "$RELEASETYPE" == "release" ]]; then
-	URL="https://github.com/istio/istio/releases/download/${RELEASE}/istio-${RELEASE}-linux.tar.gz"
-elif [[ "$RELEASETYPE" == "pre-release" ]]; then
-	URL="https://gcsweb.istio.io/gcs/istio-prerelease/prerelease/${RELEASE}/istio-${RELEASE}-linux.tar.gz"
-else
-  echo "Please specify RELEASETYPE"
-fi
+case "${OSTYPE}" in
+  darwin*)
+    if [[ "$RELEASETYPE" == "daily" ]]; then
+        URL="https://gcsweb.istio.io/gcs/istio-prerelease/daily-build/${RELEASE}/istio-${RELEASE}-osx.tar.gz"
+    elif [[ "$RELEASETYPE" == "release" ]]; then
+        URL="https://github.com/istio/istio/releases/download/${RELEASE}/istio-${RELEASE}-osx.tar.gz"
+    elif [[ "$RELEASETYPE" == "pre-release" ]]; then
+        URL="https://gcsweb.istio.io/gcs/istio-prerelease/prerelease/${RELEASE}/istio-${RELEASE}-osx.tar.gz"
```

Comment:
You can also stick all this in a common file with a function
“Get_release_url”,  or similar .

Reply:
Agree to making this into a common function.

Reply:
I have moved this part into function get_release_url() and the function is in a separate file. Thanks.